cilium-etcd-cluster/descriptors.go

```diff
@@ -61,6 +61,17 @@ func CiliumEtcdCluster(namespace, repository, version string, size int, etcdEnv
 					},
 				},
 				Affinity: &v1.Affinity{
+					NodeAffinity: &v1.NodeAffinity{
+						RequiredDuringSchedulingIgnoredDuringExecution: &v1.NodeSelector{
+							NodeSelectorTerms: []v1.NodeSelectorTerm{
+								MatchExpressions: []v1.NodeSelectorRequirement{
+									Key:      "aws.amazon.com/lifecycle",
```

Comment:
Where did you find this label? I can't find it documented anywhere.

Reply:
I'm adding it https://github.com/planetscale/planetscale-operator/pull/5140

Reply:
Ohh of course. I forgot that EKS doesn't manage workers for us.